.markdownlint.json

```diff
@@ -0,0 +1,36 @@
+{
```

Comment:
Let's use the following as a starting point for the config file:

```json
{
    "default": true,

    "blanks-around-fences": true,
    "blanks-around-headers": true,
    "fenced-code-language": true,
    "first-header-h1": {
        "level": 1
    },
    "header-increment": true,
    "header-start-left": true,
    "header-style": {
        "style": "atx"
    },
    "hr-style": {
        "style": "---"
    },
    "line-length": false,
    "list-indent": true,
    "list-marker-space": true,
    "no-alt-text": true,
    "no-bare-urls": true,
    "no-blanks-blockquote": true,
    "no-duplicate-header": true,
    "no-empty-links": true,
    "no-hard-tabs": true,
    "no-inline-html": {
        "allowed_elements": [
            "a",
            "br",
            "p"
        ]
    },
    "no-missing-space-atx": true,
    "no-multiple-blanks": true,
    "no-multiple-space-atx": true,
    "no-multiple-space-blockquote": true,
    "no-reversed-links": true,
    "no-space-in-code": true,
    "no-space-in-emphasis": true,
    "no-space-in-links": true,
    "no-trailing-punctuation": true,
    "no-trailing-spaces": false,
    "ol-prefix": {
        "style": "one_or_ordered"
    },
    "proper-names": {
        "code_blocks": false,
        "names": [
            ".NET",
            "ASP.NET",
            "JavaScript",
            "NuGet",
            "PowerShell"
        ]
    },
    "single-h1": true,
    "ul-start-left": true
}
```

Reply:
I've added you config and updated the title to reflect that these rules aren't all currently passing

Reply:
Here is the current warnings (exlucding the sample's third party files) https://gist.github.com/nschonni/9ec9b1c00fb2fa98f39f9687079e13a8